prompting-101/improving-chatGPTs-responses.md

```diff
@@ -176,6 +176,98 @@ You're likely to see a much longer response to this prompt, one which includes t
 
 </details>  
 
+**Tell the AI to provide pointers without code** 
```

Comment:
This feels a bit buried to me. What if we moved this to a new heading on par with `Instruction Prompting`?

Maybe move it to the end after all the general tips and call it something like `Learning Approaches with Prompting`?

This could let us get a bit more into why we're bringing this up and what we'd like students to get out of this more directly. We could reiterate the key points in the lesson summary as well.